Include PID, timestamp, loglevel and uniq_id on every line of logging

robinbortlik opened the pull request:
- issue: #1 
- add pid, uniq_id, timestamp and loglevel to each line of log so it is much easier to grep for specific log message when interleaving log lines happen

reevoo-samuel commented:
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **designed for change**, **easy to understand**
and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
  tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

robinbortlik commented:
@seddy can you please take a look? Thank you

Commit:
Include PID, timestamp, loglevel and uniq_id on every line of logging

- add pid, uniq_id, timestamp and loglevel to each line of log so it is much easier to grep for specific log message when interleaving log lines happen

Commit:
Bump version to 0.2.0

seddy commented on spec/grape_log_formatter/basic_spec.rb:
```diff
@@ -6,18 +6,23 @@
     it "contain time, severity and data string" do
       expect(result).to eq <<-STR
-[2016-05-06 18:20:00 +0200] INFO -- Warning message
+[2016-05-06 18:20:00 +0200] <a345dv:##{pid}> INFO -- Warning message
```
Very minor, but could be `#{uniq_id}` instead of the raw string - minor point, not really too bothered about it :).

seddy commented:
LGTM :)
